packages/discovery-provider/src/queries/get_extended_purchase_gate.py

```diff
@@ -234,7 +234,8 @@ def _get_extended_purchase_gate(
     session: Session, gate: PurchaseGate, include_network_cut=False
 ):
     price = gate.get("usdc_purchase", {}).get("price", None)
-    splits = gate.get("usdc_purchase", {}).get("splits", [])
+    original_splits = gate.get("usdc_purchase", {}).get("splits", [])
+    splits = [orig.copy() for orig in original_splits]
```

Comment:
do we want to deep copy here just in case this gets more complex?

Reply:
maybe? though that's a 20x perf hit so idk if it's worth it yet